### PR TITLE
166718951 fr skip liveness test

### PR DESCRIPTION
**Pull request by hadarbmdev**
This PR closes [#166718951](https://www.pivotaltracker.com/story/show/166718951), by removing liveness test call

* Adding 2 server config variables support:
1. skipFaceRecognition - Returns FR passed with no Zoom API interaction. default: false (env: SKIP_FACE_RECO=false)
2.allowFaceRecognitionDuplicates.  Allows passing FR process even if duplicate exists
deafult:false (env: ALLOW_FACE_RECO_DUPS=false)

### CODE REVIEW

**Comment by hadarbmdev** [comment]
Once confirmed, following msg should be pinned in protofire slack:

> Please add server .env:
> > ALLOW_FACE_RECO_DUPS=false
> SKIP_FACE_RECO=false

**Comment by sirpy** [comment]
Something is off, why does your commit include my commits?

File: src/server/verification/faceRecognition/faceRecognitionHelper.js [hunk 1]
```diff
@@ -78,7 +78,8 @@ const Helper = {
   async isLivenessPassed(zoomData: FormData) {
     try {
       let res = await ZoomClient.liveness(zoomData)
-      log.debug('liveness result:', { res })
+      let results = res.data
+      results && log.debug('liveness result:', { results })
```
Comment (sirpy) [inline src/server/verification/faceRecognition/faceRecognitionHelper.js]: results && is not required

File: src/server/verification/faceRecognition/faceRecognitionHelper.js [hunk 2]
```diff
@@ -87,11 +88,17 @@ const Helper = {
   },
 
   async isDuplicatesExist(zoomData: FormData, identifier: string) {
+    if (Config.allowFaceRecognitionDuplicates) {
+      log.info('NOTE: Skipping duplicates test')
```
Comment (sirpy) [inline src/server/verification/faceRecognition/faceRecognitionHelper.js]: write the name of the function as first parameter in log

File: src/server/verification/faceRecognition/faceRecognitionHelper.js [hunk 3]
```diff
@@ -108,8 +115,9 @@ const Helper = {
   async enroll(zoomData: FormData): EnrollResult {
     try {
       let res = await ZoomClient.enrollment(zoomData)
-      log.debug('enroll result:', { res })
-      if (res.meta.ok) return res.data
+      let results = res.data
+      results && log.debug('enroll result:', { results })
```
Comment (sirpy) [inline src/server/verification/faceRecognition/faceRecognitionHelper.js]: same no need for results && and add function name

File: src/server/verification/faceRecognition/faceRecognitionHelper.js [hunk 4]
```diff
@@ -121,7 +129,8 @@ const Helper = {
   async delete(enrollmentIdentifier: string): Promise<boolean> {
     try {
       let res = await ZoomClient.delete(enrollmentIdentifier)
-      log.debug('delete result:', { res })
+      let results = res.meta.ok
+      results && log.debug('delete result:', { results })
```
Comment (sirpy) [inline src/server/verification/faceRecognition/faceRecognitionHelper.js]: same

File: src/server/verification/faceRecognition/faceRecognitionHelper.js (hunk 1 above)
Comment (hadarbmdev) [reply src/server/verification/faceRecognition/faceRecognitionHelper.js]: removed, commit ed4ee0da4d09f3e67a4e8523866aaeddd36ed7b2, will be resolved once pushed

File: src/server/verification/faceRecognition/faceRecognitionHelper.js (hunk 2 above)
Comment (hadarbmdev) [reply src/server/verification/faceRecognition/faceRecognitionHelper.js]: fixed, commit 3a75013675688943af3b0e989cf2797881254c89, will be resolved once pushed

File: src/server/verification/faceRecognition/faceRecognitionHelper.js (hunk 4 above)
Comment (hadarbmdev) [reply src/server/verification/faceRecognition/faceRecognitionHelper.js]: removed, commit ed4ee0da4d09f3e67a4e8523866aaeddd36ed7b2, will be resolved once pushed

File: src/server/verification/faceRecognition/faceRecognitionHelper.js (hunk 3 above)
Comment (hadarbmdev) [reply src/server/verification/faceRecognition/faceRecognitionHelper.js]: removed, commit ed4ee0da4d09f3e67a4e8523866aaeddd36ed7b2, will be resolved once pushed

**Comment by sirpy** [comment]
@hadarbmdev tests are failing

**Comment by hadarbmdev** [comment]
@sirpy I fixed the failing tests

**Pull request merged**